packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+import Button from '../Button/Button';
+
+export const alertType = ['success', 'danger', 'warning', 'info'];
+
+const propTypes = {
+  type: PropTypes.oneOf(alertType).isRequired,
+  action: PropTypes.shape(Button.propTypes),
+  ariaLabel: PropTypes.string,
```

Comment:
Let's make this just label. @jgiardino  Is the label something that will change from instance to instance, or is core setting those? IE: `aria-label="Success Notification"` Downside with not allowing the user to set it would be translations.

Reply:
There are predefined strings from core for each message type that occur in two places:

- `aria-label` on `.pf-c-alert`
- screen-reader-only text that precedes the message title, for example `<span class="sr-only">Danger: </span>`
  - Note: there is currently an issue to create a class in PF4 to create our own screen-reader-only class and replace `.sr-only`: https://github.com/patternfly/patternfly-next/issues/431

These strings are intended to communicate message type to assistive devices.

That is a valid point about translations. Has there been any discussion of providing translated strings for strings defined in core? Or even at a minimum, setting up a similar structure so that when we do have predefined strings, the strings are pulled from a separate file, and not defined in the component?

Reply:
I don't think it has been talked about.  We could always revisit is hardcode those for now. We could always make an `assistiveText` prop that allows the user to set those easily if they need support for translation.